target/linux/bmips/image/bcm6318.mk

```diff
@@ -12,3 +12,17 @@ define Device/comtrend_ar-5315u
     kmod-leds-bcm6328
 endef
 TARGET_DEVICES += comtrend_ar-5315u
+
+define Device/tp-link_td-w8968-v3
+  $(Device/bcm63xx-cfe)
+  DEVICE_VENDOR := TP-Link
+  DEVICE_MODEL := TD-W8968
+  DEVICE_VARIANT := V3
+  CHIP_ID := 6318
```

Comment:
Can we add V4 as a DEVICE_ALT0_VARIANT (at least on my V4, https://github.com/openwrt/openwrt/pull/16139/ works fine)

Reply:
@wryun according to https://openwrt.org/toh/tp-link/td-w8960n_v5, the TD-W8960N v5 is the same device, so we can add it as ALT1 too.